mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
Could you elaborate so that I might investigate?  I'm unfamiliar with MuseScore's translation code, and I don't know what to search for in the files to see these usages of "pt" or "%".
The points vs. pixels situation is interesting, in that they are equivalent.  I combined them, but they could be separated in unitSuffixes and Units - though that depends on whether you want to separate them on the Page Settings dialog, which is a whole other deal.  Reviewing the places "pt" is used will illuminate all of that.